pkg/controller/disruption/disruption.go

```diff
@@ -688,6 +688,12 @@ func (dc *DisruptionController) getExpectedScale(pdb *policy.PodDisruptionBudget
 	return
 }
 
+// IsPodHealthy determines if an individual pod will be considered in
+// the health count of a PDB.
+func IsPodHealthy(pod *v1.Pod) bool {
```

Comment:
Could do this in a separate patch, if necessary.

Reply:
No logic change, but wrapping the underlying implementation in a public function so it can be consumed in the eviction API.  This way if the healthy calculation for PDBs were to change in the future to something other than just 'podutil.IsPodReady(pod)' it will be picked up in those places.